Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/photo_field.tsx

```diff
@@ -0,0 +1,653 @@
+      font: '400 16px Roboto'
+    },
+    closeIcon: {
+      width: '20px',
+      height: '20px',
+      cursor: 'pointer' as 'pointer'
+    },
+    buttonBoxSmall: {
+      boxSizing: 'border-box' as 'border-box',
+      display: 'flex' as 'flex',
+      flexDirection: 'column' as 'column',
+      flexWrap: 'wrap' as 'wrap',
+      alignItems: 'center' as 'center',
+      justifyContent: 'space-between' as 'space-between',
+      height: '86px'
+    },
+    buttonBoxLarge: {
+      display: 'flex' as 'flex',
+      flexDirection: 'row' as 'row',
+      flexWrap: 'wrap' as 'wrap',
+      justifyContent: 'space-between' as 'space-between',
+      alignItems: 'center' as 'center'
+    },
+    placeholderImage: {
+      position: 'relative' as 'relative',
+      height: '24px',
+      width: '30px',
+      top: 'calc(50% - 12px)',
+      left: 'calc(50% - 15px)'
+    },
+    imageSmall: {
+      objectFit: 'cover' as 'cover',
+      height: '100%',
+      width: '100%'
+    },
+    imageLarge: {
+      objectFit: 'cover' as 'cover',
+      height: '100%',
+      width: '100%'
+    },
+    imageBoxSmall: {
+      boxSizing: 'border-box' as 'border-box',
+      height: '166px',
+      width: '248px',
+      overflow: 'hidden' as 'hidden',
+      borderRadius: '1px',
+      border: '1px solid #EBEBEB',
+      backgroundColor: '#F8F8F8'
+    },
+    imageBoxLarge: {
+      boxSizing: 'border-box' as 'border-box',
+      height: '216px',
+      width: '324px',
+      overflow: 'hidden' as 'hidden',
+      borderRadius: '1px',
+      border: '1px solid #EBEBEB',
+      backgroundColor: '#F8F8F8'
+    },
+    hiddenInput: {
+      width: '0.1px',
+      height: '0.1px',
+      opacity: 0,
+      overflow: 'hidden' as 'hidden',
+      position: 'absolute' as 'absolute',
+      zIndex: -1
+    }
+  };
+  private static readonly SPECIAL_STYLE = StyleSheet.create({
+    button: {
+      boxSizing: 'border-box' as 'border-box',
+      cursor: 'pointer' as 'pointer',
+      display: 'flex' as 'flex',
+      flexDirection: 'row' as 'row',
+      flexWrap: 'wrap' as 'wrap',
+      justifyContent: 'center' as 'center',
+      alignItems: 'center' as 'center',
+      minWidth: '153px',
+      maxWidth: '248px',
+      height: '34px',
+      backgroundColor: '#684BC7',
+      color: '#FFFFFF',
+      font: '400 14px Roboto',
+      border: '1px solid #684BC7',
+      borderRadius: '1px',
+      outline: 0,
+      ':active': {
+        backgroundColor: '#4B23A0'
+      },
+      ':hover': {
+        backgroundColor: '#4B23A0'
+      }
+    }
+  });
+  private static readonly HEADER_TEXT = 'Change Picture';
+  private static readonly BROWSE_BUTTON_TEXT = 'BROWSE';
+  private static readonly SUBMIT_BUTTON_TEXT = 'SUBMIT';
+  private static readonly PADDING = '18px';
+  private static readonly PADDING_BETWEEN_ELEMENTS = '30px';
+}
+
+interface SliderProperties {
+
+  /** Callback that updates the value */
+  onChangeValue?: (num: number) => void;
```

Comment:
Rename this to ```onChange``` to be consistent with HTML naming conventions. There are occasions where we would be more specific and name it ```onValueChange``` but only if we needed to disambiguate two different values that could change.